tcms/core/tests/test_middleware.py

```diff
@@ -16,3 +23,23 @@ def test_unapplied_migrations(self):
             'container">documentation</a>'
         response = self.client.get('/', follow=True)
         self.assertContains(response, unapplied_migration_message)
+
+
+class TestCsrfDisableMiddleware(unittest.TestCase):
+    def test_prcess_view(self):
+        request = HttpRequest()
+        middleware = CsrfDisableMiddleware()
+        middleware.process_view(request,
+                                _callback=None,
+                                _callback_args=None,
+                                _callback_kwargs=None)
+        self.assertEqual(request._dont_enforce_csrf_checks, True)
```

Comment:
This middleware should be removed and thus doesn't need tests, see https://github.com/kiwitcms/Kiwi/issues/297

Reply:
I have removed the test. :smile: